docs/coordinates/index.rst

```diff
@@ -207,7 +207,7 @@ a similar quick way to get an `~astropy.coordinates.EarthLocation`::
     <EarthLocation (-1463969.3018517173, -5166673.342234327, 3434985.7120456537) m>
 
 To see the list of site names available, use
-:func:`astropy.coordinates.EarthLocation.get_site_names`.
+the :class:`~astropy.coordinates.EarthLocation` :func:`astropy.coordinates.EarthLocation.get_site_names` method.
```

Comment:
I chose the original way just because the novice user who hasn't seen static/class methods might get confused by this. So that might be better?  But I could go either way.